### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PatrikDurdevic**

### CODE REVIEW

**Comment by PatrikDurdevic** [comment]
Work in progress.. I need to still create a google colab demo

File: README.md [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+<i>Trane Demos</i>
+</p>
+
+This repository contains demos that showcase the functionality in the [Trane library](https://github.com/trane-dev/Trane).
+
 # Usage
 
-## Google CoLab
+## Google Colab
+Open Google Colaboratory Demo Link from the list below
```
Comment (gsheni) [inline README.md:14]:
```suggestion
Open Google Cola link from the list below:
```

File: README.md [hunk 2]
```diff
@@ -12,8 +26,26 @@ git clone https://github.com/trane-dev/trane-demos
...
+```bibtex
+@inproceedings{schreck2016would,
+  title={What Would a Data Scientist Ask? Automatically Formulating and Solving Predictive Problems},
+  author={Schreck, Benjamin and Veeramachaneni, Kalyan},
+  booktitle={Data Science and Advanced Analytics (DSAA), 2016 IEEE International Conference on},
+  pages={440--451},
+  year={2016},
+  organization={IEEE}
+}
+```
```
Comment (gsheni) [inline README.md:51]: Only needed in Trane repo
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,21 @@
-# Automated Prediction Engineering with Trane
+<p align="center">
+<img width=30% src="https://dai.lids.mit.edu/wp-content/uploads/2018/06/Trane-logo-300x180.jpg" alt=“Trane_Logo” />
+</p>
+
+<p align="center">
+<i>Trane Demos</i>
+</p>
+
+This repository contains demos that showcase the functionality in the [Trane library](https://github.com/trane-dev/Trane).
```
Comment (gsheni) [inline README.md:9]:
```suggestion
This repository contains demos that showcase the functionality of [Trane](https://github.com/trane-dev/Trane).
```

**Review by gsheni — COMMENTED** [review]
Looks good overall. Just some minor suggestions

File: README.md (hunk 1 above)
Comment (PatrikDurdevic) [reply README.md:14]:
```suggestion
Open Google Colab link from the list below:
```

**Comment by PatrikDurdevic** [comment]
@gsheni Suggestions look good, thank you, feel free to merge

**Review by gsheni — APPROVED** [review]

**Pull request merged**